qtpy/QtGui.py

```diff
@@ -108,6 +156,12 @@ def movePositionPatched(
     QMouseEvent.globalPosition = lambda self: __QPointF(
         float(self.globalX()), float(self.globalY()))
 if PYQT6 or PYSIDE6:
+    QMouseEvent.pos = lambda self: self.position().toPoint()
+    QMouseEvent.x = lambda self: self.position().toPoint().x()
+    QMouseEvent.y = lambda self: self.position().toPoint().y()
+    QMouseEvent.globalPos = lambda self: self.globalPosition().toPoint()
+    QMouseEvent.globalX = lambda self: self.globalPosition().toPoint().x()
+    QMouseEvent.globalY = lambda self: self.globalPosition().toPoint().y()
```

Comment:
```suggestion
```

This block appears to have been accidentally retained in the merge, as it duplicates the added block below.